File: VB/CustomSeriesPointDrawingSample/Model/Order.vb
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema

Namespace CustomSeriesPointDrawingSample.Model

    Public Partial Class Order

        <Key>
        <Required>
        <Column("OrderID")>
        Public Property OrderId As Integer

        <ForeignKey("Employee")>
        <Column("EmployeeID")>
        Public Property EmployeeId As Integer?

        Public Property OrderDate As Date

        <Column(TypeName:="smallmoney")>
        Public Property Freight As Decimal?

        Public Overridable Property Employee As Employee
    End Class
End Namespace

File: CS/CustomSeriesPointDrawingSample/Model/Order.cs
namespace CustomSeriesPointDrawingSample.Model {
    using System;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;

    public partial class Order
    {
        [Key]
        [Required]
        [Column("OrderID")]
        public int OrderId { get; set; }


        [ForeignKey("Employee")]
        [Column("EmployeeID")]
        public int? EmployeeId { get; set; }

        public DateTime OrderDate { get; set; }

        [Column(TypeName = "smallmoney")]
        public decimal? Freight { get; set; }

        public virtual Employee Employee { get; set; }
    }
}
